Phase1/test/FullTextSearchTest.java

```diff
@@ -0,0 +1,49 @@
+package test;
+
+import src.*;
+import java.util.*;
+import static org.junit.Assert.*;
+import org.junit.Test;
+import org.junit.Before;
+
+public class FullTextSearchTest {
+    private InvertedIndex invertedIndex;
+    private DocsFileReader docsFileReader;
+    private FilterHandler filterHandler;
+    private Tokenizer tokenizer;
+    private PlusFilter plusFilter;
+    private MinusFilter minusFilter;
+    private NoSignFilter noSignFilter;
+
+    @Before
+    public void initialize() {
+        invertedIndex = new InvertedIndex();
+        tokenizer = new Tokenizer();
+        docsFileReader = new DocsFileReader();
+        plusFilter = new PlusFilter();
+        noSignFilter = new NoSignFilter();
+        minusFilter = new MinusFilter();
+        filterHandler = new FilterHandler(invertedIndex, plusFilter, noSignFilter, minusFilter);
+    }
+
+    @Test
+    public void testFullTextSearch() {
+        initialize();
```

Comment:
implement this method with @Before annotation cause that call it automatically before run test so this line is useless